dwave/system/samplers/dwave_sampler.py

```diff
@@ -94,8 +94,8 @@ class DWaveSampler(dimod.Sampler, dimod.Structured):
             Switch to a new QPU in the rare event that the currently connected
             system goes offline. Note that different QPUs may have different
             hardware graphs and a failover will result in a regenerated
-            :attr:`.nodelist`, :attr:`.edgelist`, :attr:`.properties` and
-            :attr:`.parameters`.
+            :attr:`DWaveSampler.nodelist`, :attr:`DWaveSampler.edgelist`, :attr:`DWaveSampler.properties` and
```

Comment:
If we specify DWaveSampler here then classes that inherit will no link correctly. In this case it is OK because it's the class's docstring, but in general using the local namespace is preferred.

Reply:
Changed back.